ansible/roles/run-test-target/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Log into quay.io
+  shell:
+    cmd: "docker login -u {{ quay_username }} --password-stdin quay.io"
+    stdin: "{{ quay_password }}"
```

Comment:
Out of curiosity, have we considered using the [docker_login module](https://docs.ansible.com/ansible/latest/collections/community/docker/docker_login_module.html#ansible-collections-community-docker-docker-login-module) instead of running `docker login` in a shell?

Reply:
That is an excellent idea, I'll see what I can do.

Reply:
As discussed offline this is painful to support due to python version differences (particularly on RHEL 7) so we'll keep it as it is for now.

I'll probably figure something out and fix in a follow up PR